server/activate.go

```diff
@@ -103,11 +103,16 @@ func (p *Plugin) OnActivate() error {
 		}()
 	}
 
-	rtcServer, err := rtc.NewServer(rtc.ServerConfig{
-		ICEPortUDP:      *cfg.UDPServerPort,
-		ICEHostOverride: cfg.ICEHostOverride,
-		ICEServers:      rtc.ICEServers(cfg.getICEServers()),
-	}, newLogger(p), p.metrics.RTCMetrics())
+	rtcServerConfig := rtc.ServerConfig{
+		ICEPortUDP:                       *cfg.UDPServerPort,
+		ICEHostOverride:                  cfg.ICEHostOverride,
+		ICEServers:                       rtc.ICEServers(cfg.getICEServers(false)),
+		TURNCredentialsExpirationMinutes: *cfg.TURNCredentialsExpirationMinutes,
+	}
+	if *cfg.ServerSideTURN {
+		rtcServerConfig.TURNStaticAuthSecret = cfg.TURNStaticAuthSecret
+	}
```

Comment:
Should we add text to the settings page that `changing this setting will require a server restart`?

Reply:
Great point. I actually had a similar thoughts when implementing this. I think we should do a review and document any setting (there are a few) that require a restart, a plugin restart that is (no need to restart the Mattermost server) but I guess that's what you meant. Will open a ticket to address this.